src/services/usercentrics-service.ts

```diff
@@ -26,7 +26,9 @@ const update = () => listeners.forEach((s) => s(state))
 const subscribe = (x: (x: ServiceState) => void): (() => void) => {
     listeners.push(x)
     return () => {
-        listeners.indexOf(x) > -1 ? listeners.splice(listeners.indexOf(x), 1) : undefined
+        if (listeners.includes(x)) {
+            listeners.splice(listeners.indexOf(x), 1)
+        }
```

Comment:
This got flagged by ESLint and looks a bit strange; since the intention is to remove the specific listener from the list, it's best to do a conditional call rather than unassigned expression.

> Expected an assignment or function call and instead saw an expression
https://typescript-eslint.io/rules/no-unused-expressions/